iscsi/osfamilymap.yaml

```diff
@@ -33,46 +33,63 @@ Debian:
 
 RedHat:
   config:
+    name:
+     tgtd: /etc/tgt/targets.conf     #cent6
     servicename:
       open-iscsi: iscsi
       isns: isnsd
+      tgtd: tgtd                     #cent6
   isns:
     pkgs:
       wanted:
         - isns-utils
         - yum-plugin-versionlock
+        # target-isns                #fedora29
   initiator:
     pkgs:
       wanted:
-        - iscsi-initiator-utils
-        - iscsi-initiator-utils-iscsiuio
-        - libiscsi
-        - libiscsi-utils
+        - iscsi-initiator-utils               #cent6/cent7/fedora/amazon
+           {%- if grains.osmajorrelease >= 7 %}
+        - iscsi-initiator-utils-iscsiuio      #cent7/fedora
+        - libiscsi                            #cent7/fedora
+        - libiscsi-utils                      #cent7/fedora
+           {%- endif %}
   target:
     pkgs:
       wanted:
-        - libvirt-daemon-driver-storage-iscsi
-        - netbsd-iscsi
-        - udisks2-iscsi
-        - yum-plugin-versionlock
-        - targetcli
-        - device-mapper-persistent-data
-        # scsi-target-utils
-        # fcoe-target-utils
+	- device-mapper-persistent-data                #cent6/cent7/fedora/amazon
```

Comment:
Be careful to use consistent indentation.

Reply:
Got it - thanks @n-rodriguez